api/app/Policies/PoolCandidatePolicy.php

```diff
@@ -51,68 +85,94 @@ public function create(User $user)
     {
-        return $user->isAdmin();
+        return  $user->id === $poolCandidate->user_id && $user->isAbleTo("submit-own-application");
     }
 
     /**
-     * Determine whether the user can restore the model.
+     * Determine whether the user can archive the model.
      *
      * @param  \App\Models\User  $user
      * @param  \App\Models\PoolCandidate  $poolCandidate
      * @return \Illuminate\Auth\Access\Response|bool
      */
-    public function restore(User $user, PoolCandidate $poolCandidate)
+    public function archive(User $user, PoolCandidate $poolCandidate)
     {
-        return $user->isAdmin();
+        return $user->id === $poolCandidate->user_id && $user->isAbleTo("archive-own-submittedApplication");
     }
 
     /**
-     * Determine whether the user can permanently delete the model.
+     * Determine whether the user can suspend the model.
      *
      * @param  \App\Models\User  $user
      * @param  \App\Models\PoolCandidate  $poolCandidate
      * @return \Illuminate\Auth\Access\Response|bool
      */
-    public function forceDelete(User $user, PoolCandidate $poolCandidate)
+    public function suspend(User $user, PoolCandidate $poolCandidate)
     {
-        return false;
+        return $user->id === $poolCandidate->user_id && $user->isAbleTo("suspend-own-submittedApplication");
     }
 
     /**
-     * Determine whether the user can archive an application
+     * Determine whether the user can count the number of items in the model.
+     *
+     * Note: Everyone needs to be able to count applicants
+     * for now
      *
-     * @param  \App\Models\User  $user
      * @param  \App\Models\PoolCandidate  $poolCandidate
      * @return \Illuminate\Auth\Access\Response|bool
      */
-    public function archiveApplication(User $user, PoolCandidate $poolCandidate)
+    public function count()
     {
-        return $user->isAdmin() || $user->id === $poolCandidate->user_id;
+        return true;
     }
 
     /**
-     * Determine whether the user can delete an application
+     * Determine whether the user can delete the model.
      *
      * @param  \App\Models\User  $user
      * @param  \App\Models\PoolCandidate  $poolCandidate
      * @return \Illuminate\Auth\Access\Response|bool
      */
-    public function deleteApplication(User $user, PoolCandidate $poolCandidate)
+    public function delete(User $user, PoolCandidate $poolCandidate)
     {
-        return $user->isAdmin() || $user->id === $poolCandidate->user_id;
+        return $user->id === $poolCandidate->user_id && $user->isAbleTo("delete-own-draftApplication");
+    }
+
+    /**
+     * Determine if User owns the Pool for the PoolCandidate
+     *
+     * @param  \App\Models\User  $user
+     * @param  \App\Models\PoolCandidate  $poolCandidate
+     * @return \App\Models\Team|bool
+     */
+    private function userBelongsToPoolCandidatePool(?User $user, PoolCandidate $poolCandidate)
```

Comment:
I still think this function is unnecessary.

Reply:
Sorry, just clarifying what you mean by unnecessary.

1. Because it is only used once so isn't worth abstracting out into its own method
2. Because we don't actually care about checking the team

Reply:
2, because we don't need to check the team in the function like that.

In policies, we should never be manually checking whether a user is part of a team. We should simply pass the relevant team into the `isAbleTo` function, and check that way.

Reply:
Make sense, removed in [remove check if user is part of team in policy](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5815/commits/735584e5c1d333fb476fb997a5eb56871257296f)